### ISSUE TITLE
initContainers

### ISSUE THREAD

**Issue by kfox1111** [metallb/metallb#1512]
Trying to migrate from the bitnami chart to this one. I'm using the initContainers feature of the bitnami chart to work around:
https://github.com/metallb/metallb/issues/277

with something like:
```
speaker:
   initContainers:
    - name: arp
      command:
      - /bin/sh
      - -ec
      - |
        ip link set dev ovsbr0 arp off;
      securityContext:
        capabilities:
            add:
             - NET_ADMIN
```

This chart needs support for this.

**Comment by fedepaol** [comment]
Hi @kfox1111 , would you mind adding a reference to the bitnami chart and how the template look like?
cc @gclawes

**Comment by kfox1111** [comment]
Chart:
https://github.com/bitnami/charts/tree/master/bitnami/metallb
template section:
https://github.com/bitnami/charts/blob/master/bitnami/metallb/templates/speaker/daemonset.yaml#L63-L65

**Comment by fedepaol** [comment]
Closing the issue since #277 is now merged and released, and there's no need for the workaround.

**Comment by samip5** [comment]
> Closing the issue since #277 is now merged and released, and there's no need for the workaround.

This workaround is still useful, at least until a new release is cut.

**Comment by fedepaol** [comment]
The release was cut, the interface selector is available in 0.13.7
